Lib/ufo2ft/featureCompiler.py

```diff
@@ -92,6 +95,10 @@ def __init__(self, ufo, ttFont=None, glyphSet=None, **kwargs):
 
         glyphOrder = ttFont.getGlyphOrder()
         if glyphSet is not None:
+            if set(glyphOrder) != set(glyphSet.keys()):
+                print("Glyph order incompatible")
```

Comment:
This should probably be a `logger.error`.

Reply:
this is just before an `assert` so it doesn't matter much if we `print` or do `logger.error`, an AssertionError should never trigger and is not something client can expect to recover from. It's just the error message for the hard crash, it could be moved to the `assert` optional string parameter, if you want.